tests/ops/conftest.py

```diff
@@ -60,7 +60,6 @@
 from .fixtures.saas.square_fixtures import *
 from .fixtures.saas.stripe_fixtures import *
 from .fixtures.saas.twilio_conversations_fixtures import *
-from .fixtures.saas.zendesk_fixtures import *
```

Comment:
In order to make this dynamic, I moved the fixtures into the task test file. We can go with this or figure out a way to do dynamic imports.